docs/plugins/repository-s3.asciidoc

```diff
@@ -304,6 +304,8 @@ You may further restrict the permissions by specifying a prefix within the bucke
 The bucket needs to exist to register a repository for snapshots. If you did not create the bucket then the repository
 registration will fail.
 
+Note: All bucket operations are using the path style access pattern. The DNS style access pattern is never used.
```

Comment:
I think the AWS terminology is "virtual hosted style"

Reply:
Right changing :)